subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -292,7 +292,7 @@ abstract class InHouseInstrumentationTestRunner :
             report.registerIncident(AppCrashException(incident))
             report.reportTestCase()
         } catch (t: Throwable) {
-            logger.warn("Can't register and report unexpected incident", t)
+            logger.critical("Can't register and report unexpected incident", t)
```

Comment:
I think an incident report fail is not critical because this is not lead to test failures

Reply:
why so? i thought if we haven't report here test is probably lost

Reply:
I thought the incident means that test is ERROR. But If it's LOST behavior why we simple handle that better to avoid a LOST

Reply:
We're trying to make it an error, sending incident, but if something unexpected happens (e.g. this catch closure), we need to know about it and handle somehow

That's why this message promoted to critical;
plan is to look for this criticals and fix it's reasons